static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -93,7 +102,7 @@ export function InviteBanner({missingMembers, onSendInvite, organization}: Props
       <MemberCardContent>
         <MemberCardContentRow>
           <IconGithub size="sm" />
-          {/* TODO: create mapping from integration to lambda external link function */}
+          {/* TODO(cathy): create mapping from integration to lambda external link function */}
           <StyledExternalLink href={`http://github.com/${member.externalId}`}>
             {tct('@[externalId]', {externalId: member.externalId})}
```

Comment:
can't be translated

Reply:
should i do `{t('@')}{member.externalId}` instead?

Reply:
just `@{member.externalId}` since nothing should be translated there